_data/harnesses/connect-sink-timestamp/ksql.yml

```diff
@@ -0,0 +1,92 @@
+            file: tutorials/connect-sink-timestamp/ksql/markup/answer/short-answer.adoc
+
+dev:
+  steps:
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/connect-sink-timestamp/ksql/markup/dev/init.adoc
+
+    - title: Get Confluent Platform
+      content:
+        - action: make_file
+          file: docker-compose.yml
+          render:
+            file: tutorials/connect-sink-timestamp/ksql/markup/dev/make-docker-compose.adoc
+
+        - action: execute_async
+          file: tutorial-steps/dev/docker-compose-up.sh
+          render:
+            file: tutorials/connect-sink-timestamp/ksql/markup/dev/start-compose.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/wait-for-containers.sh
+          render:
+            skip: true
+
+    - title: Write the program interactively using the CLI
+      content:
+        - action: docker_ksql_cli_session
+          container: ksqldb-cli
+          docker_bootup_file: tutorial-steps/dev/start-cli.sh
+          stdout:
+            directory: tutorial-steps/dev/outputs
+          column_width: 75
+          render:
+            file: tutorials/connect-sink-timestamp/ksql/markup/dev/start-cli.adoc
+          stdin:
+            - file: tutorial-steps/dev/set-properties_offset.sql
+              render:
+                file: tutorials/connect-sink-timestamp/ksql/markup/dev/set-properties_offset.adoc
+
+            - file: tutorial-steps/dev/create-stream.sql
+              render:
+                file: tutorials/connect-sink-timestamp/ksql/markup/dev/create-stream.adoc
+
+            - file: tutorial-steps/dev/insert-data.sql
+              render:
+                file: tutorials/connect-sink-timestamp/ksql/markup/dev/insert-data.adoc
+
+            - file: tutorial-steps/dev/describe-raw-stream.sql
+              render:
+                file: tutorials/connect-sink-timestamp/ksql/markup/dev/describe-raw-stream.adoc
+
+            - file: tutorial-steps/dev/select-from-stream.sql
+              render:
+                file: tutorials/connect-sink-timestamp/ksql/markup/dev/select-from-stream.adoc
+
+            - file: tutorial-steps/dev/create-sink-connector.sql
+              render:
+                file: tutorials/connect-sink-timestamp/ksql/markup/dev/create-sink-connector.adoc
+
+            - file: tutorial-steps/dev/show-connector.sql
+              render:
+                file: tutorials/connect-sink-timestamp/ksql/markup/dev/show-connector.adoc
+        - action: sleep
+          ms: 5000
+          render:
+            skip: true
+
+    - title: Query the data in Postgres
+      content:
+        - action: execute
+          file: tutorial-steps/dev/query-sink.sh
+          stdout: tutorial-steps/dev/outputs/query-sink-output.log
+          render:
+            file: tutorials/connect-sink-timestamp/ksql/markup/dev/query-sink.adoc
```

Comment:
Not sure how consistent the tutorials are on this point, but WDYT about adding a Cleanup step, e.g. http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/DEVX-2488/connect-add-key-to-source/ksql.html#clean-up

Reply:
I think that's a clean way to end things. I added it in my last commit d22ac26